.gitignore

```diff
@@ -33,3 +33,4 @@ lerna-debug.log*
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+.env
```

Comment:
If you add file to gitignore git will not detect changes in that file but that file will be exist in repository. If you want to remove file from git, you should remove file, commit that change and after it add that file to gitignore.

Only if I right...

Reply:
Fixed. You were right.